.gitignore

```diff
@@ -1,4 +1,4 @@
 node_modules
 dist/
 .cache/
-*.env
\ No newline at end of file
+*.env.development
```

Comment:
`.env.local` is more idiomatic for a local file. Is there a reason we're not using it?